antarest/desktop/systray_app.py

```diff
@@ -182,5 +182,9 @@ def run_systray_app(config_file: Path) -> None:
     wait_for_server_start()
     notification_popup("Antares Web Server started, you can manage the application within the system tray.")
     open_app()
-    systray_app.app.exec_()
-    server.kill()
+    try:
+        systray_app.app.exec_()
+    except:  # catch QT error
```

Comment:
I would remove the except so that we still get the stack trace (the exception will be raised after the "finally" clause execution